op-node/p2p/peer_scorer.go

```diff
@@ -61,22 +61,23 @@ func NewScorer(cfg *rollup.Config, peerStore Peerstore, metricer ScoreMetrics, a
 // The returned [pubsub.ExtendedPeerScoreInspectFn] is called with a mapping of peer IDs to peer score snapshots.
 // The incoming peer score snapshots only contain gossip-score components.
 func (s *scorer) SnapshotHook() pubsub.ExtendedPeerScoreInspectFn {
-	blocksTopicName := blocksTopicV1(s.cfg)
```

Comment:
We were hardcoding an old topic here. This PR future proofs us as it handles all topics which may be in use for peer scoring.